examples/ERC20/ERC20.sol

```diff
@@ -4,24 +4,24 @@ contract WARP {
     uint8  public decimals    = 18;
     uint256 public totalSupply= 100000000000000000000000000000000000;
 
-    mapping (address => uint)                       public  balanceOf;
-    mapping (address => mapping (address => uint))  public  allowance;
+    mapping (uint => uint)                       public  balanceOf;
```

Comment:
Just curious, how does it help?

Reply:
It removes the code that does the unnecessary shifting to get the first 20 bytes.